chainer_chemistry/models/relgcn.py

```diff
@@ -79,14 +82,16 @@ def __init__(self, out_dim=64, hidden_channels=None, n_update_layers=None,
         self.input_type = input_type
         self.scale_adj = scale_adj
 
-    def __call__(self, x, adj):
+    def __call__(self, batch):
```

Comment:
can we keep backward compatibility?

Reply:
use NumpyTupleDataset for dense data in chemistry dataset, and use wrapper Chain for dense SNS dataset example??

Reply:
fixed!